dist/cta-button/cta-button.css

```diff
@@ -105,25 +105,6 @@ a.cta-btn--large {
   font-size: var(--font-size-medium);
   min-height: 48px;
 }
-a.cta-btn--large-truncated {
```

Comment:
How come we're removing truncation for large CTA buttons? Other smaller buttons have truncation and we have at least one story with truncation. If we're not keeping the truncation, we should probably remove the story too. 
<img width="522" alt="image" src="https://github.com/eBay/skin/assets/1675667/47976f02-487e-4fd8-99b8-2b8d06473b46">

Reply:
Okay, I will remove this.

Reply:
This was marked as deprecated since we should use `cta-btn--truncated` for both small and large. See https://github.com/eBay/skin/pull/1933

The story should still stay because it is using the new truncated syntax.